DictionaryTypes.txt

```diff
@@ -305,6 +314,13 @@ enum(role) "Values for 'ref_role'; the role of a reference"
 }
 
 
+enum(next) "File pointers"
+{
+    none	-1   "None"
+    seq	1   "Sequence"
+}
```

Comment:
In MNE we have `FIFFV_NEXT_SEQ` and `FIFFV_NEXT_NONE` that we always use when writing tags. For example this is how we write a coordinate transform:
```
def write_coord_trans(fid, trans):
    """Write a coordinate transformation structure."""
    data_size = 4 * 2 * 12 + 4 * 2
    fid.write(np.array(FIFF.FIFF_COORD_TRANS, dtype='>i4').tostring())
    fid.write(np.array(FIFF.FIFFT_COORD_TRANS_STRUCT, dtype='>i4').tostring())
    fid.write(np.array(data_size, dtype='>i4').tostring())
    fid.write(np.array(FIFF.FIFFV_NEXT_SEQ, dtype='>i4').tostring())
    # Write the actual transform bytes according to the `trans` def
   ...
```
And the only time `NEXT_NONE` is used is in:
```
def write_nop(fid, last=False):
    """Write a FIFF_NOP."""
    fid.write(np.array(FIFF.FIFF_NOP, dtype='>i4').tostring())
    fid.write(np.array(FIFF.FIFFT_VOID, dtype='>i4').tostring())
    fid.write(np.array(0, dtype='>i4').tostring())
    next_ = FIFF.FIFFV_NEXT_NONE if last else FIFF.FIFFV_NEXT_SEQ
    fid.write(np.array(next_, dtype='>i4').tostring())
```
... so should `next` be a new `enum` type like this? Or should it live somewhere else in the spec?

Reply:
FIFFV_NEXT_SEQ == 0!

Reply:
Is 1 used for anything else? This makes me suspect we have been writing the wrong value for quite some time!

Reply:
This could be a typo?  MEGIN fiff-library has always defined FIFFV_NEXT_SEQ=0, please check what is the actual value in the MNE code?

Reply:
It is correct in MNE CPP but not MNE Python

https://github.com/mne-tools/mne-python/blob/master/mne/io/constants.py#L162

I'll fix it there and in this PR.